dto/api_input.py

```diff
@@ -72,34 +79,41 @@ class TeamsStatisticsOrPlayersDTO(BaseDTO):
     name = fields.Str(required=False, missing=None)
 
 
-class SearchDTO(BaseDTO):
-    leagues__sport_id = fields.Int(required=False, missing=None)
-    countries__country_id = fields.Int(required=False, missing=None)
-    letter = fields.Str(required=False, missing="")
+class PaginationDTO(BaseDTO):
     page = fields.Int(required=False, missing=0)
     per_page = fields.Int(required=False, missing=0)
 
+class FilterDTO(BaseDTO):
+    name = fields.Str(required=True)
+    value = fields.Raw(required=True)
+
+class SearchDTO(BaseDTO):
+    filters = fields.List(fields.Nested(FilterDTO), required=False)
+    pagination = fields.Nested(PaginationDTO, many=False)
+    order = fields.Str(required=False, missing=None)
```

Comment:
Why are `order` and `field` in SearchDTO and not in FilterDTO?

Also, why do we need `order` in the first place? We should have a separate filter for that

Reply:
We can have a generic order_by filter in our base DAL an use it